README.md

```diff
@@ -1,5 +1,28 @@
 # Gonum [![Build Status](https://travis-ci.org/gonum/gonum.svg?branch=master)](https://travis-ci.org/gonum/gonum) [![Coverage Status](https://coveralls.io/repos/gonum/gonum/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/gonum?branch=master) [![GoDoc](https://godoc.org/gonum.org/v1/gonum?status.svg)](https://godoc.org/gonum.org/v1/gonum) [![Go Report Card](https://goreportcard.com/badge/github.com/gonum/gonum)](https://goreportcard.com/report/github.com/gonum/gonum)
 
+## Installation
+
+The core packages of the gonum suite are written in pure Go with some assembly.
+Installation is done using `go get`.
+```
+go get -u gonum.org/v1/gonum/...
+```
+
+## Build tags
```

Comment:
I just wanted to get the information out there. I'm happy for it to go somewhere else in the longer term, and some of these may go away in the future anyway.